practice-two/src/components/FormValidation/FormValidation.jsx

```diff
@@ -105,37 +100,37 @@ const FormValidation = ({
     }
   }, [selectedCustomer]);
 
+  useEffect(() => {
+    const isFormEmpty = Object.values(formData).some((value) => value === '');
```

Comment:
no need to create state for invalid empty data just check render the button

Reply:
![image](https://github.com/datdragon777/React-Training/assets/74806826/a619e97e-3c6e-4dc5-bcce-34e83889d640)